x-pack/plugins/reporting/public/hacks/job_completion_notifier.js

```diff
@@ -38,13 +40,19 @@ uiModules.get('kibana')
       const reportObjectTitle = job._source.payload.title;
       const reportObjectType = job._source.payload.type;
 
-      const isJobSuccessful = get(job, '_source.status') === 'completed';
+      const isJobSuccessful = get(job, '_source.status') === JobStatuses.JOB_STATUS_COMPLETED;
 
       if (!isJobSuccessful) {
         const errorDoc = await jobQueueClient.getContent(job._id);
         const text = errorDoc.content;
         return toastNotifications.addDanger({
-          title: `Couldn't create report for ${reportObjectType} '${reportObjectTitle}'`,
+          title: (
+            <FormattedMessage
+              id="xpack.reporting.jobCompletionNotifier.error.couldnotCreateReportTitle"
```

Comment:
Maybe do not use .error namespace
Also couldnotCreateReportTitle => couldNotCreateReportTitle